driver-ui/src/main/java/eu/tsystems/mms/tic/testframework/webdrivermanager/WebDriverCapabilityLogHelper.java

```diff
@@ -71,12 +71,14 @@ private Map<String, Object> shortChromeExtensionStrings(Map<String, Object> capa
         if (chromeOptionsObject != null) {
             final Map chromeOptions = (Map) chromeOptionsObject;
             if (chromeOptions.containsKey("extensions")) {
-                chromeOptions.put("extensions", shortAllStringsInLists(chromeOptions.get("extensions")));
+                List<String> extensions = shortAllStringsInLists(chromeOptions.get("extensions"));
+                if (extensions.size() > 0) chromeOptions.put("extensions", extensions);
```

Comment:
This change seems to be ineffectual: If the list from `get` has been empty, the result would also have been empty, if it had been missing, the method `shortAllStringsInLists` would crash.